packages/node-sdk/src/config.ts

```diff
@@ -0,0 +1,9 @@
+import { version } from "../package.json";
+
+export const API_HOST = "https://tracking.bucket.co";
+export const SDK_VERSION_HEADER_NAME = "bucket-sdk-version";
+export const SDK_VERSION = version;
```

Comment:
It might actually also help us to send the sdk name as an HTTP header, so we can break down usage even further. Right now we can just see a version, but won't know if it came from js, react, node or whatever.

Reply:
added the name of the SDK